resource/ui/ts/models/timeseries.ts

```diff
@@ -43,68 +44,183 @@ module Models {
         /**
          * QueryResultSet matches the successful output of the /ts/query
          * endpoint. This needs to be kept in sync with the
-         * TimeSeriesQueryResult protobuffer message on the server.
+         * TimeSeriesQueryResponse protobuffer message on the server.
          * (/protos/timeseries.proto).
          */
         export interface QueryResultSet {
```

Comment:
My comment seems to have moved to the wrong place.
Even better. Let's drop those interfaces into their own module Proto.
And even better than that, +1 to Tamir's suggestion.